### PR TITLE
feat: add preview of email inbox

### PR DESCRIPTION
**Pull request by davidsoderberg**
### What change does this PR introduce? 
Adding previews of email inbox

### Other information (Screenshots)
Screenshot 2022-11-20 at 16 41 41
Screenshot 2022-11-20 at 16 41 48
Screenshot 2022-11-20 at 16 41 54

### CODE REVIEW

**Comment by davidsoderberg** [comment]
Screenshot 2022-11-20 at 16 53 17

File: apps/web/src/components/templates/email-editor/EmailContentCard.tsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ import { VariableManager } from '../VariableManager';
 import { useIntegrations } from '../../../api/hooks';
 import { Grid, useMantineTheme } from '@mantine/core';
 import { format } from 'date-fns';
```
Comment (ainouzgali) [inline apps/web/src/components/templates/email-editor/EmailContentCard.tsx:12]: unused import? 
Loved that you extracted all that content

File: apps/web/src/pages/templates/editor/PreviewMobileInbox/InboxItem.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { Skeleton } from '@mantine/core';
+import { colors } from '../../../../design-system';
+
+export const ItemSkeleton = () => (
+  <div
+    style={{
+      marginLeft: '15px',
+      marginRight: '15px',
+      borderTop: `2px solid ${colors.B20}`,
```
Comment (ainouzgali) [inline apps/web/src/pages/templates/editor/PreviewMobileInbox/InboxItem.tsx:9]: They are the same colors for light and dark color scheme?

File: apps/web/src/pages/templates/editor/Preview.tsx [hunk 3]
```diff
@@ -13,6 +15,7 @@ export const Preview = ({ activeStep }: { activeStep: number }) => {
   const contentType = watch(`steps.${activeStep}.template.contentType`);
   const htmlContent = watch(`steps.${activeStep}.template.htmlContent`);
   const editorContent = watch(`steps.${activeStep}.template.content`);
+  const preheader = watch(`steps.${activeStep}.template.preheader`);
```
Comment (ainouzgali) [inline apps/web/src/pages/templates/editor/Preview.tsx:18]: In general (we have quite a few places) we need to look at the other watch options - like a callback or the useWatch hook. As this way I think it triggers unnecessary re-renders.

**Review by ainouzgali — APPROVED** [review]
Amazing. So excited about this. 
I think we should go for the tab view between mobile and web that we talked about, wdyt? Also we should add hot keys/ shortcuts to navigate between editor and preview

**Pull request merged**